app/views/authors/edit.html.haml

```diff
@@ -0,0 +1,2 @@
+%h1 Edit author
+= render "authors/form", author: @author
```

Comment:
oh, well you actually passed author as a local var, but decided not to use it :D